mep/common/migrations/0007_add_data_viewer_group.py

```diff
@@ -0,0 +1,88 @@
+# Generated by Django 2.2.11 on 2020-03-11 18:30
+
+from django.contrib.auth.management import create_permissions
+from django.db import migrations
+
+data_viewer_perms = {
+    'accounts': [
+        'view_account',
+        "view_address",
+        # 'view_accountaddress',
+        'view_borrow',
+        'view_event',
+        'view_purchase',
+        'view_reimbursement',
+        'view_subscription',
+        'view_subscriptiontype'
+    ],
+    'books': [
+        'view_publisher',
+        'view_publisherplace',
+        'view_creator',
+        'view_creatortype',
+        'view_work',
+        'view_edition',
+        'view_editioncreator',
+    ],
+    "footnotes": [
+        "view_bibliography",
+        "view_footnote",
+        "view_sourcetype"
+    ],
+    "people": [
+        "view_country",
+        "view_infourl",
+        "view_person",
+        "view_profession",
+        "view_relationship",
+        "view_relationshiptype"
+    ],
+    'djiffy': [
```

Comment:
the use of single/double quotes in this data structure seems inconsistent (presumably because it got copied from somewhere and edited?)